DataFormats/L1TParticleFlow/interface/HPSPFTau.h

```diff
@@ -67,6 +67,7 @@ namespace l1t {
     float sumNeutralIso() const { return sumNeutralIso_; }
     float sumCombinedIso() const { return sumCombinedIso_; }
     float sumChargedIsoPileup() const { return sumChargedIsoPileup_; }
+    float Z() const { return Z_; }
```

Comment:
Class members and methods should have lower case initials
```suggestion
    float z() const { return z_; }
```